<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>960</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>960</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>960</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Launcher</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: url(&quot;E:/Tugas dan kerjaan/rafi/bg960.png&quot;);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>790</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andalus</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(245, 223, 186);</string>
    </property>
    <property name="text">
     <string>Play!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>640</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andalus</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(245, 223, 186);</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andalus</family>
      <pointsize>16</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : white;</string>
    </property>
    <property name="text">
     <string>Ver.</string>
    </property>
   </widget>
   <widget class="QLabel" name="versionLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>570</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andalus</family>
      <pointsize>16</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : white;</string>
    </property>
    <property name="text">
     <string>0.0.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
